src/display.rs

```diff
@@ -88,12 +88,48 @@ impl From<renderer::Error> for Error {
     }
 }
 
+pub struct VisualBell {
+    /// Visual bell duration
+    duration: u16,
+
+    /// The last time the visual bell rang, if at all
+    last_rang: Option<::std::time::Instant>,
```

Comment:
Bikeshed - maybe calling this `start` or `start_time` would be more appropriate?